data/actions/scripts/other/potions.lua

```diff
@@ -1,23 +1,68 @@
-local ultimateHealthPot = 8473
-local greatHealthPot = 7591
-local greatManaPot = 7590
-local greatSpiritPot = 8472
-local strongHealthPot = 7588
-local strongManaPot = 7589
-local healthPot = 7618
-local manaPot = 7620
-local smallHealthPot = 8704
-local antidotePot = 8474
-local greatEmptyPot = 7635
-local strongEmptyPot = 7634
-local emptyPot = 7636
-
+--- CONFIGURATION ---
+potions = {
+	removePots = true,
+	addEmptyPots = true, -- Only works if removepots isn't false or nil
+	words = "Aaaah...",
+	[8473] = { -- ultimate health potion
+		vocations = {4},
+		minLevel = 130,
+		health = {650, 850},  -- min, max
+		empty = 7635
+	},
+	[7591] = { -- great health potion
+		vocations = {4},
+		minLevel = 80,
+		health = {425, 575},
+		empty = 7635
+	},
+	[7590] = { -- great mana potion
+		vocations = {1, 2},
+		minLevel = 80,
+		mana = {150, 250},
+		empty = 7635
+	},
+	[8472] = { -- great spitit potion
+		vocations = {3},
+		minLevel = 80,
+		health = {250, 350},
+		mana = {100, 200},
+		empty = 7635
+	},
+	[7588] = { -- strong health potion
+		vocations = {3, 4},
+		minLevel = 50,
+		health = {250, 350},
+		empty = 7634
+	},
+	[7589] = { -- strong mana potion
+		vocations = {1, 2, 3},
+		minLevel = 50,
+		mana = {115, 185},
+		empty = 7634
+	},
+	[7618] = { -- health potion
+		health = {75, 125},
+		empty = 7636
+	},
+	[7620] = { -- mana potion
+		mana = {125, 175},
```

Comment:
This is actually the healing range for the potion of id **7618** *(health potion)*.